src/core/router/routes.tsx

```diff
@@ -31,6 +33,22 @@ export const routerConfig: AppRouterConfig = {
       view: <AboutPage toHomePage={() => goTo('')} />,
       showForNewUsers: true,
     },
+    {
+      slug: 'privacy',
+      title: i18n.t('modes.privacy'),
+      icon: <Reports16 />,
+      view: <PrivacyPage />,
+      parentRoute: 'about',
+      visibility: 'always',
+    },
+    {
+      slug: 'cookies',
+      title: 'modes.cookies',
+      icon: <Reports16 />,
+      view: <CookiesPage />,
+      parentRoute: 'about',
+      visibility: 'never',
```

Comment:
`visibility` here relates to visibility in sidebar (you can still see the view of the route by the link), may be smth like `isInNavigation` or `isInSidebar`/`inSidebar` is more accurate

Reply:
The router is a higher-level entity, the sidebar is what displays it. I don't think the sidebar should be mentioned here, it may not be there at all, or there may be more than one

Reply:
yeah, i agree it's not about Sidebar itself, that's why i proposed an option about general navigation. In the end this property defines whether the route is visible in any apps navigation